src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,139 @@
+import { Link, useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { useEffect, useState } from 'react';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { Loader } from './Loader';
+
+export const PeoplePage: React.FC = () => {
+  const [loading, setLoading] = useState(false);
+  const [people, setPeople] = useState<Person[]>([]);
+  const [error, setError] = useState(false);
+
+  useEffect(() => {
+    setLoading(true);
+
+    getPeople()
+      .then((value) => {
+        setPeople(value);
+      })
+      .catch(() => {
+        setError(true);
+      })
+      .finally(() => {
+        setLoading(false);
+      });
+  }, []);
+  const { slug } = useParams();
```

Comment:
move it to the top of your component. keep it near with states and initial constants